app/assets/javascripts/components/LandingPage.jsx

```diff
@@ -1,13 +1,27 @@
 class LandingPage extends React.Component {
   render() {
+  	const { Grid } = ReactBootstrap;
+  	const { Row } = ReactBootstrap;
+  	const { Col } = ReactBootstrap;
     return (
-    	<div>
-	    	<img 
-	    		src="http://socialjusticecollaborative.org/wp-content/uploads/2015/09/OpenSansSemibold.jpg"
+	  	<Row bsClass="row custom-row">
+	  		<Col sm={9} md={9} lg={9}>
+	  			<img 
+	    		src="http://socialjusticecollaborative.org/wp-content/uploads/2015/09/Social-Justice-Collaborative-San-Fransisco-Another-Look-05.13.16-2.jpg"
 	    		className="landing_banner"
-	    	/>
-				<h1> social justice boys </h1>
-			</div>
+	    		height={775}
+	    		width={1000}
+	    		/>
+	      </Col>
+	      <Col sm={2} md={2} lg={2} className="home-logo">
+	      	<img 
+	    		src="http://socialjusticecollaborative.org/wp-content/uploads/2017/04/logo-large.png"
```

Comment:
You committed the image file for the logo too. You should use it instead of a link